doc/Makefile

```
@@ -45,13 +48,23 @@ clean:
 	rm -rf generated/*
 	rm -rf modules/*
 
+sync_git:
+	if [[ -d !  $(BENCH_LOGREG_PATH) ]]; then
+	    git clone https://github.com/benchopt/benchmark_logreg_l2 $(BENCH_LOGREG_PATH) > /dev/null 2>&1
+	else
+	    pushd $(BENCH_LOGREG_PATH)
+	    git pull
+	    popd
+	fi
```

Pour n'utiliser que des command compatible avec `sh`, je crois que tu peux mettre
```suggestion
	if [ -d !  $(BENCH_LOGREG_PATH) ]; then
	    git clone https://github.com/benchopt/benchmark_logreg_l2 $(BENCH_LOGREG_PATH) > /dev/null 2>&1
	else
	    (cd $(BENCH_LOGREG_PATH); git pull)
	fi
```